linker/jvm/src/main/scala/org/scalajs/linker/backend/closure/ClosureLinkerBackend.scala

```diff
@@ -73,18 +73,26 @@ final class ClosureLinkerBackend(config: LinkerBackendImpl.Config)
    *  @param unit [[standard.LinkingUnit LinkingUnit]] to emit
    *  @param output File to write to
    */
-  def emit(unit: LinkingUnit, output: LinkerOutput, logger: Logger)(
+  def emit(moduleSet: ModuleSet, outputSpec: OutputSpec, logger: Logger)(
       implicit ec: ExecutionContext): Future[Unit] = {
-    verifyUnit(unit)
+    verifyModuleSet(moduleSet)
+
+    require(moduleSet.modules.size == 1,
+        "Cannot user multiple modules with the Closure Compiler")
```

Comment:
No. The renaming across the modules needs to be consistent. So it might be possible to make this work, but it seems the main use case we want to address here is to feed these things downstream to webpack or other. So IMO we should not try to support this in a first iteration (probably provide better error messages though :)).